web/client/components/geostory/layouts/sections/Banner.jsx

```diff
@@ -0,0 +1,166 @@
+    storyTheme,
+    mediaViewer,
+    contentToolbar,
+    inView,
+    bubblingTextEditing = () => {}
+}) => {
+    const hideContent = get(focusedContent, "target.id") === contentId;
+    const visibility = hideContent ?  'hidden' : 'visible';
+    const expandableBackgroundClassName = expandableMedia && background && background.type === 'map' ? ' ms-expandable-background' : '';
+    const deletePath = path.replace('.background', '');
+
+    return (
+        <section
+            ref={inViewRef}
+            className={`ms-section ms-section-banner${expandableBackgroundClassName}`}
+            id={id}
+        >
+            <ContainerDimensions>
+                {({ height }) =>
+                // when section height is less then the view height
+                // background height need to be equal to section size
+                // this is important when working with z-index of section
+                // in case we increase the z-index of title the whole background is visible and overlap next section
+                    <Background
+                        { ...background }
+                        // selector used by sticky polyfill to detect scroll events
+                        scrollContainerSelector="#ms-sections-container"
+                        key={background.id}
+                        mode={mode}
+                        id={contentId}
+                        path={path}
+                        update={updateBackground}
+                        add={add}
+                        editMedia={editMedia}
+                        expandable={expandableMedia}
+                        updateSection={updateSection}
+                        cover={cover}
+                        remove={{removeFunc: remove, customPath: deletePath}}
+                        width={viewWidth}
+                        sectionType={sectionType}
+                        backgroundPlaceholder={{
+                            background: `url(${cover ? coverPattern : titlePattern })`,
+                            backgroundSize: `${cover ? 64 : 600 }px auto`
+                        }}
+                        tools={{
+                            [MediaTypes.IMAGE]: ['editMedia', 'cover', 'fit', 'size', 'align', 'theme', 'remove'],
+                            [MediaTypes.MAP]: ['editMedia', 'cover', 'editMap', 'size', 'align', 'theme', 'remove'],
+                            [MediaTypes.VIDEO]: ['editMedia', 'cover', 'fit', 'size', 'align', 'theme', 'muted', 'autoplay', 'loop', 'remove']
+                        }}
+                        height={height >= viewHeight
+                            ? viewHeight
+                            : height}
+                        storyTheme={storyTheme}
+                        mediaViewer={mediaViewer}
+                        contentToolbar={contentToolbar}
+                        inView={inView}
+                    />}
+            </ContainerDimensions>
+            <SectionContents
```

Comment:
The banner section does not need SectionContents component because it has only a background without text contents.
Instead to use the `SectionContents` component as a plecaholder to set the correct height of the section we could replace it with a `div` and remove all the logic that does not belong to the banner.

eg:
```js
<div
    style={{
        minHeight: cover ? viewHeight : 300,
        pointerEvents: 'none'
    }}>
</div>
```